docs/pages/apis/schema.mdx

```diff
@@ -144,18 +144,18 @@ export default config({
         isHidden: ({ session }) => false,
         hideCreate: ({ session }) => false,
         hideDelete: ({ session }) => false,
-        listView: {
-          defaultFieldMode: ({ session }) => 'read',
-          initialColumns: ['name', /* ... */],
-          initialSort: { field: 'name', direction: 'ASC' },
-          pageSize: 50,
-        },
         createView: {
           defaultFieldMode: ({ session }) => 'edit',
         },
         itemView: {
           defaultFieldMode: ({ session, item }) => 'edit',
         },
+        listView: {
+          defaultFieldMode: ({ session }) => 'read',
+          initialColumns: ['name', /* ... */],
+          initialSort: { field: 'name', direction: 'ASC' },
```

Comment:
unrelated to this pr, @mitchellhamilton can you take consistency here into account with the graphql spec you're looking at right now?